src/web/lib/components/PresetThemeSelector/index.js

```diff
@@ -2,21 +2,21 @@ import React from "react";
 
 import BrowserPreview from "../BrowserPreview";
 
-import { presetColors } from "../../../../lib/constants";
+import { presetThemes } from "../../../../lib/themes";
 import Metrics from "../../../../lib/metrics";
 
 import "./index.scss";
 
 export const PresetThemeSelector = ({ setTheme }) => (
   <div className="preset-theme-selector">
     <h2>Choose a preset theme</h2>
-    {presetColors.map((colors, themeId) => {
+    {presetThemes.map((colors, themeId) => {
       const theme = {
         images: "",
         colors
       };
       return (
-        <div key={themeId} className="preset-theme-preview">
+        <div key={themeId} title={colors.title} className="preset-theme-preview">
```

Comment:
Punted on adding the title to the UI in any way other than as a hover-title

Reply:
LGTM. I tried hacking a title to add an `onmouseover=""` handler in the theme JSON, but looks like React automatically escapes that XSS nonsense:

```
"title": "Panini\" onmouseover=\"alert(1)",
```